Trello/trello2csv.sh

```diff
@@ -16,59 +16,79 @@ usage () {
 	echo "   If you pass a list to exclude, it will not be created in the csv"
 	echo
 }
-[ "$1" == "-h" ] && usage && exit 1
-[ $# -lt 2 ] && usage && exit 1
+[[ "$1" == "-h" ]] && usage && exit 1
+[[ $# -lt 2 ]] && usage && exit 1
 
-# Trello API
```

Comment:
I'd say this whole part can be removed. TRELLO_KEY and TRELLO_TOKEN MUST be env vars, so it might be useful to source an external file other than putting things directly in the script.

Reply:
Added environment file `trello2csv.env`
Unset variables in script (but left a reference to environment file).
Changed README.md with instructions for env file - fixing also result order.